docs/anchoring-assets/anchor-platform/events/integration.mdx

```diff
@@ -0,0 +1,140 @@
+      retries: 0
+      linger_ms: 1000
+      batch_size: 10
+      poll_timeout_seconds: 10
+```
+
+</CodeExample>
+
+Anchor Platform allows a subset of the Kafka producer's client configuration to be set. The following are the supported configuration options:
+
+- `bootstrap_server`: A comma-separated list of Kafka brokers in the format `host:port`. This corresponds to the Kafka producer's `bootstrap.servers` configuration.
+- `client_id`: The client ID to be used by the Kafka producer. This corresponds to the Kafka producer's `client.id` configuration.
+- `retries`: The number of times the Kafka producer will retry sending a message. This corresponds to the Kafka producer's `retries` configuration.
+- `linger_ms`: The time in milliseconds the Kafka producer will wait before sending a batch of messages. This corresponds to the Kafka producer's `linger.ms` configuration.
+- `batch_size`: The maximum number of messages the Kafka producer will send in a single batch. This corresponds to the Kafka producer's `batch.size` configuration.
+
+For more information on the Kafka producer's client configuration, see the [Kafka documentation](https://kafka.apache.org/documentation/#producerconfigs).
+
+## Receiving Events
+
+The event service can be used to send events to client and business server callback endpoints. The event service will send events to these endpoints as HTTP POST requests with the event data in the request body.
+
+### As a Client Application
+
+Client applications can receive updates whenever the `status` of a SEP transaction changes.
+
+To receive events as a client application, you will need to expose a callback URL that the event service can send events to. The event service will send a POST request to this endpoint with the event data in the request body.
+
+Anchor Platform will only send events to clients listed in the client configuration. See the client configuration documentation for more information.
+
+#### Callback Signing
+
+Anchor Platform signs the callback requests it sends to client applications. The signature is included in the `Signature` header of the request. The callback URL signature specification can be found in the corresponding SEP protocol specifications.
+
+- [SEP-0006](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#url-callback-signature)
+- [SEP-0024](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#url-callback-signature)
+- [SEP-0031](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0031.md#url-callback-signature)
+
+### As a Business Server
+
+In addition to SEP transaction status updates, business servers can receive events about SEP-31 quote creation or SEP-12 customer information updates. The schema of the event data will depend on the type of event being sent. Visit the [Event API documentation](/api/anchor-platform/callbacks/post-event) for more information about the schema of the event data.
```

Comment:
When do we send event for SEP-12 customer updated?

Reply:
On every SEP-12 customer PUT. I'll update it.

Reply:
Don't we already receive a callback on SEP-12 PUT to the business server? Why do we want to also receive an event associated with it?

Reply:
I was just curious why we added it in a first place. Got it, thanks!